gapic-libraries-bom/pom.xml

```diff
@@ -672,6 +672,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.shopping</groupId>
+        <artifactId>google-shopping-merchant-reports-bom</artifactId>
```

Comment:
I don't think this should go into the libraries bom, since it's not a cloud library,.
@suztomo

Reply:
Good catch.

@mpeddada1 Would you remove the artifacts under com.google.shopping from the BOM?

Reply:
Can we modify the script such that group IDs that are not `com.google.cloud` do not get added to the bom automatically?

Reply:
Yes, thanks for pointing this out! Will remove this from the BOM